stoobly_agent/app/cli/helpers/openapi_endpoint_adapter.py

```diff
@@ -73,19 +73,32 @@ def adapt(self, spec: Spec) -> List[EndpointShowResponse]:
           endpoint: EndpointShowResponse = {}
           endpoint['id'] = endpoint_counter
           endpoint['method'] = http_method.upper()
-          endpoint['host'] = parsed_url.netloc
+          endpoint['host'] = 'localhost' if parsed_url.netloc == '' else parsed_url.netloc
 
           joined_path = self.__urljoin(parsed_url.path, path_name)
           split_parts = joined_path.split('/')
           pattern_path = []
+          segment_names = []
           for part in split_parts:
             sanitized_part = part
+            segment_name = part
             if part.startswith('{') and part.endswith('}'):
               sanitized_part = '%' 
+              segment_name = f":{part[1:-1]}"
             pattern_path.append(sanitized_part)
+            segment_names.append(segment_name)
           pattern_path_str = '/'.join(pattern_path)
           endpoint['match_pattern'] = pattern_path_str
           endpoint['path'] = joined_path
+
+          endpoint['path_segment_names'] = []
+          for segment_name in segment_names:
+            if segment_name == "":
+              continue
+            path_component_name: RequestComponentName = {}
+            path_component_name['name'] = segment_name
+            path_component_name['type'] = "Alias" if segment_name.startswith(':') else "Static"
```

Comment:
Does the capitalization for Alias and Static matter?

Reply:
I don't think so. I probably capitalized them because I was just copying what was in the payload in the network tab. Changed it to all lowercase and path segment section looked correct after importing so should be fine.